patches/master_patch.patch

```diff
@@ -414,83 +414,104 @@ index a5ea92f2878459b49f88d4895c7d18f2e157dd6c..9cffca0352713bce12b4035c02612cd7
      }
    }
  
+diff --git a/chrome/browser/resource_coordinator/tab_lifecycle_unit.cc b/chrome/browser/resource_coordinator/tab_lifecycle_unit.cc
+index 1b5026face136534578062e4d3c9c0c955dadd20..b8d7284641b79fc9d473f09885f3d232276b2073 100644
+--- a/chrome/browser/resource_coordinator/tab_lifecycle_unit.cc
++++ b/chrome/browser/resource_coordinator/tab_lifecycle_unit.cc
+@@ -7,10 +7,12 @@
+ #include "base/logging.h"
+ #include "base/metrics/histogram_macros.h"
+ #include "base/process/process_metrics.h"
++#include "chrome/browser/browser_process.h"
+ #include "chrome/browser/media/webrtc/media_capture_devices_dispatcher.h"
+ #include "chrome/browser/media/webrtc/media_stream_capture_indicator.h"
+ #include "chrome/browser/profiles/profile.h"
+ #include "chrome/browser/resource_coordinator/tab_lifecycle_observer.h"
++#include "chrome/browser/resource_coordinator/tab_manager.h"
+ #include "chrome/browser/resource_coordinator/time.h"
+ #include "chrome/browser/ui/tabs/tab_strip_model.h"
+ #include "content/public/browser/favicon_status.h"
+@@ -221,14 +223,9 @@ bool TabLifecycleUnitSource::TabLifecycleUnit::Discard(
+       GetWebContents()->NeedToFireBeforeUnload());
+ 
+   content::WebContents* const old_contents = GetWebContents();
+-  content::WebContents::CreateParams create_params(tab_strip_model_->profile());
```

Comment:
please use `MUON_CHROMIUM_BUILD` for block comment out

Reply:
The code was moved to `CreateNullContents()` more or less verbatim